_rules/95e043.md

```diff
@@ -0,0 +1,202 @@
+---
+name: Auto-play audio has a control mechanism
+test_type: atomic
+
+description: |
+  This rule checks that each auto-play audio or video element has a control mechanism.
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+- Audio output
+
+authors:
+- Anne Thyme Nørregaard
+- Bryn Anderson
+---
+
+## Applicability
+
+This rule applies to any HTML `<audio>` `<video>` or `<source>` `src` file content with a `duration` of more than 3 seconds, that is set to `autoplay` and or `loop`, and has audio output that is not `paused` or `muted`.
```

Comment:
You write 

> and has audio output that is not `paused` or `muted`.

For the composite rule you are a little more specific since you explicitly talk about the attribute : 

>  and that has either a `paused` or `muted` attribute equal to `false`

I suggest : 

> This rule applies to any HTML `<audio>` `<video>` or `<source>` `src` file content with a `duration` of more than 3 seconds, that is set to `autoplay` and or `loop`, and that has either a `paused` or `muted` attribute equal to `false`.

Reply:
I would use the same applicability as the composite rule